test/jdk/ProblemList.txt

```diff
@@ -835,3 +835,9 @@ java/awt/Checkbox/CheckboxNullLabelTest.java 8340870 windows-all
 java/awt/dnd/WinMoveFileToShellTest.java 8341665 windows-all
 java/awt/Menu/MenuVisibilityTest.java 8161110 macosx-all
 java/awt/Modal/NativeDialogToFrontBackTest.java 7188049 windows-all,linux-all
+
+############################################################################
+
+# jdk_since_checks
+
+tools/sincechecker/modules/java.base/JavaBaseCheckSince.java 8355746 generic-all
```

Comment:
```suggestion
tools/sincechecker/modules/java.base/JavaBaseCheckSince.java 8358627 generic-all
```

@nizarbenalla Per our offline discussion, I created [JDK-8358627](https://bugs.openjdk.org/browse/JDK-8358627) to track this.